docs/CNVs/CNV-methods.tex

```diff
@@ -482,6 +497,135 @@ \subsection{Using Panel of Normals for Allelic Fraction Model} \label{allelic-Po
+%
+\begin{align}
+\prod_t {\rm Poisson}(n_{st} | d_s c_{st} \lambda_{st}) \approx {\rm N}(\vb_s | \vm_s, \vSigma_s), \, m_{st} = \ln(n_{st}/(c_{st} d_s)), \, \vSigma_{st} = 1/n_{st}
+\end{align}
+%
+to obtain the joint likelihood
+%
+\begin{equation}
+P(\vn, \vb, \vz | \vc, \vW, \vm, \vPsi, d ) = \prod_s {\rm N}(\vz_s | \vzero, \vI) {\rm N}(\vb_s | \vW \vz_s + \vm, \vPsi) {\rm N}(\vb_s | \vm_s, \vSigma_s)
+\end{equation}
+%
+
+We can integrate out $\vb$ via the identity $\int {\rm N}(x | \mu_1, \sigma_1^2) {\rm N}(x | \mu_2, \sigma_2^2) \, dx = {\rm N}(\mu_1 | \mu_2, \sigma_1^2 + \sigma_2^2)$ to obtain
+%
+\begin{equation}
+P(\vn, \vz | \vc, \vW, \vm, \vPsi, d, \vn) = \prod_s {\rm N}(\vz_s | \vzero, \vI) {\rm N}(\vm_s | \vW \vz_s + \vm, \vPsi + \vSigma_s)
+\end{equation}
+%
+
+For fixed $\vc$ we may obtain maximum likelihood estimates of $\vW$, $\vSigma$, and $\vm$ by an iterative EM approach in which we alternate between computing the Gaussian posteriors of each $\vz_s$ (E step) and maximizing the log-likelihood with respect to $\vW$, $\vSigma$, and $\vm$ (M step).  This is similar to the factor analysis model discussed in Chapter 12 of Bishop.  We note that a second level of iteration, on $\vc$, allows us to correct for CNV events that may be present in the panel of normals: alternate between 1) learning $\vW$, $\vSigma$, and $\vm$ and 2) calling $\vc_s$ for each sample in the panel.
+
+The E step follows from substituting $\vPsi \rightarrow \vPsi_s \equiv  \vPsi + \vSigma_s$ in Bishop's Equations 12.66-67 to get
+%
+\begin{align}
+\vG_s =& \left( \vI + \vW^T \vPsi_s^{-1} \vW \right)^{-1} \\
+E \left[ \vz_s \right] =& \vG_s \vW^T \vPsi_s^{-1} (\vm_s - \vm) \\
+E \left[ \vz_s \vz_s^T \right] =& \vG_s + E[\vz_s] E[\vz_s]^T
+\end{align}
+%
+
+For the M steps, we must maximize with respect to 
+%
+\begin{align}
+E \left[ \sum_s \ln {\rm N}(\vm_s | \vW \vz_s + \vm, \vPsi + \vSigma_s) \right] =& -\frac{1}{2} \sum_{st} \left[ \ln \Psi_{st}+ E[(\vW \vz_s + \vm - \vm_s)_t^2] (\Psi_{st})^{-1} \right] 
+\end{align}
+which reduces to
+\begin{equation}
+ -\frac{1}{2} \sum_{st} \left[ \ln (\vPsi_t + \Sigma_{st}) + \left( \vW_t  E[\vz_s \vz_s^T] \vW_t^T + (\vm_t - \vm_{st})^2 +2 (\vm_t - \vm_{st}) \vW_t E[\vz_s] \right)\Psi_{st}^{-1} \right]
+\end{equation}
+%
+
+
+The M step for $\vm$ is routine and comes out to:
+\begin{align}
+\vm = \left( \sum_s \vPsi_{s}^{-1} \right)^{-1} \sum_s \left[ \vPsi_{s}^{-1}  (\vW  E[\vz_{s}] - \vm_{s} ) \right] 
+\end{align}
+
+The M step for $\vPsi_t$ is
+\begin{align}
+\sum_s \frac{1}{\vPsi_t + \Sigma_{st}} - \frac{\left( \vW_t  E[\vz_s \vz_s^T] \vW_t^T + (\vm_t - \vm_{st})^2 +2 (\vm_t - \vm_{st}) \vW_t E[\vz_s] \right)}{(\vPsi_t + \Sigma_{st})^2} = 0,
+\end{align}
+which is straightforward to solve numerically.  Alternatively, we could approximate $\vSigma_s \approx \bar{\vSigma} \equiv \frac{1}{S} \sum_s \vSigma_s$, in which case we obtain
```

Comment:
Since we are working with raw read counts, \Sigma_s will be inversely proportional to the average read depth of the sample in question. If d_s changes significantly from sample to sample (does it?), averaging over s may not be a good idea. I suggest using Eq. (55) as a guess, followed by a few Brent iterations until convergence of Eq. (54).